src/index.js

```diff
@@ -0,0 +1,70 @@
+import './style.css';
+import add from './add.js';
+import { event, adde } from './edit.js';
+import remove from './remove.js';
+import check from './module/completed.js';
+import clearC from './module/clearCompleted.js';
+
+let task = [];
+if (localStorage.getItem('used') === null) {
+  task = [];
+} else {
+  task = JSON.parse(localStorage.getItem('used'));
+}
+
+const flex = document.querySelector('.list');
+
+for (let i = 0; i < task.length; i += 1) {
+  flex.innerHTML += `<div class="flex-list"><input class="checker" type="checkbox"><div class="value-list"> ${task[i].discription} </div> <a class="right" href="#"><i class="fa-sharp fa-solid fa-ellipsis-vertical"></i></a><p class="index">${i + 1}</p></div>`;
+}
+
+const checkInp = document.querySelectorAll('.checker');
```

Comment:
Kindly, Declare all the _**Variables**_ on the top of the codes It is best practice for _Clean Code_ _(Optional But Highly Recommended)_